src/Elliptic/DiscontinuousGalerkin/DgOperator.hpp

```diff
@@ -469,7 +469,7 @@ struct DgOperatorImpl<System, Linearized, tmpl::list<PrimalFields...>,
                 Spectral::needs_projection(face_mesh, mortar_mesh, mortar_size)
                     ? boundary_data.project_to_mortar(face_mesh, mortar_mesh,
                                                       mortar_size)
-                    : std::move(boundary_data);
+                    : boundary_data;
```

Comment:
I think the `std::move` is fine and you can `NOLINT` it, perhaps with the following comment:

When no projection is necessary we can safely move the boundary data from the face, because we only have a single neighbor in this direction.